Libraries/LibWeb/CSS/Parser/ValueParsing.cpp

```diff
@@ -360,19 +360,20 @@ RefPtr<CSSStyleValue> Parser::parse_family_name_value(TokenStream<ComponentValue
         }
 
         if (peek.is(Token::Type::Ident)) {
-            auto ident = tokens.consume_a_token().token().ident();
-
-            // CSS-wide keywords are not allowed
-            if (is_css_wide_keyword(ident))
-                return nullptr;
+            auto maybe_custom_ident = parse_custom_ident(tokens, {});
+            if (!maybe_custom_ident.has_value()) {
+                tokens.discard_a_token(); // ident
```

Comment:
Why does this discard a token? It was already consumed by parse_custom_ident(). (Or at least it should be.)

Reply:
My understanding is that the token doesn't end up getting consumed in `parse_cusom_ident()` if the token isn't a valid `<custom-ident>` because the transaction it starts isn't committed.

Reply:
Oh yeah I shouldn't code review just after waking up.
New question, why consume the token and continue on failure? We used to return null in that case.

Reply:
> why consume the token and continue on failure?

That's definitely not correct. My latest change makes sure we collect all identifiers that make up a family-name into the `parts` list - we then perform the standard custom-ident checks against the complete string.

Doing this correctly leads to an extra 8 subtest passes :tada: